packages/@coorpacademy-components/src/molecule/base-modal/test/fixtures/with-footer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {convert} from 'css-color-function';
+import ButtonLink from '../../../../atom/button-link';
+
+const primarySkinColor = '#0061FF';
+const buttonCancelProps = {
+  onClick: () => console.log('cancel'),
+  label: 'Cancel'
+};
+const buttonConfirmProps = {
```

Comment:
Wondering if we should abstract in the component the cancel/confirm logic ? 💭 

`cancel=true || {callback?, label?, iconName?, color?}`, same for `confirm`